### ISSUE TITLE
Jobs in PMO project

### ISSUE THREAD

**Issue by g4s8** [zerocracy/farm#692]
All new jobs assigned to me appears in agenda as `PMO` jobs:
![scrot-2018-03-02-10 52 44](https://user-images.githubusercontent.com/1524988/36888781-156df5fe-1e08-11e8-9b50-733bcfb506aa.png)

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#692](https://github.com/zerocracy/farm/issues/692) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
com.zerocracy.farm.reactive.StkGroovy[125] java.lang.IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/staff/elections/invalidate_all_elections.groovy
sun.reflect.GeneratedMethodAccessor144[-1] java.lang.reflect.InvocationTargetException: null
com.amazonaws.http.AmazonHttpClient$RequestExecutor[1113] com.amazonaws.SdkClientException: Unable to execute HTTP request: null
sun.nio.ch.FileChannelImpl[110] java.nio.channels.ClosedChannelException: null
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm/tree/1.0-SNAPSHOT) /cc @yegor256

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 points just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by yegor256** [comment]
@g4s8 fixed in 1642c26

**Comment by 0crat** [comment]
The job [#692](https://github.com/zerocracy/farm/issues/692) is now out of scope
